### ISSUE TITLE
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xd7 in position 30: invalid continuation byte

### ISSUE THREAD

**Issue by Bietola** [davidavdav/audacity.py#7]
I get the error while using it for a `aup3` file (maybe that's the problem...), the full command is:

```
python -m audacity inp.aup3 out.wav
```

while running in a venv.

**Comment by adabru** [comment]
I think this project is using aup2 as aup3 has switched to sqlite3 format. You can use this tool instead: https://github.com/audacity/audacity-project-tools.

**Comment by davidavdav** [comment]
Yes, sorry, I haven't used audacity in ages.  If they've changed the format then this package has probably become obsolete, but there is a `audacity-project-tools` that apparently fixes everything you needed to get fixed with audacity.
